temporal/api/cloud/sink/v1/message.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package temporal.api.cloud.sink.v1;
+
+option go_package = "go.temporal.io/api/cloud/sink/v1;sink";
+
+import "google/protobuf/timestamp.proto";
+
+message ExportSinkSpec {
```

Comment:
Where do users apply the sink message to a Namespace?  It doesn't appear to be used in either the service or namespace protos?  Please make sink an option when creating a namespace and provide users the ability to add a sink to a namespace at a later time.